src/redux/rootSaga.js

```diff
@@ -1,7 +1,8 @@
 import { all } from 'redux-saga/effects';
 import newsSaga from './news/news.sagas';
 import categoriesSaga from './categories/categories.sagas';
+import newsDetailSaga from './news-detail/news-detail.sagas';
 
 export default function* rootSaga() {
-  yield all([newsSaga(), categoriesSaga()]);
+  yield all([newsSaga(), categoriesSaga(), newsDetailSaga()]);
```

Comment:
In my opinion it is better to merge sagas related to news list and news details in one file. We should usually have one saga per business entity

Reply:
done